src/main/webapp/onelive/functions.js

```diff
@@ -1,3 +1,40 @@
-// Write custom js functions for the onelive page here
-
+//function to load upcoming events
+function loadUpcomingEvents() {
+    $.ajax({
+        type: 'GET',
+        url: 'https://sef-dataholder.herokuapp.com/calendar/onelive?maxResults=3',
+        dataType: 'json',
+        success: function (payload) {
+            if (payload.items === undefined || payload.items.length == 0) {
+                $("#section-events").hide();
```

Comment:
Shall we display a friendly message to inform the user that there are no listed events, instead of hiding?